ChangeLog

```diff
@@ -1,3 +1,17 @@
+2015-01-01  Luiz Irber <irberlui@msu.edu>
+
+    * lib/hllcounter.{cc,hh}: Implemented a HyperLogLog counter.
+	* khmer/{_khmermodule.cc, __init__.py}: added HLLCounter class
```

Comment:
our standard in ChangeLog is to use spaces, not tabs, and to indent to the '*' in subsequent lines.